plasma_framework/contracts/src/framework/registries/ExitGameRegistry.sol

```diff
@@ -27,6 +27,15 @@ contract ExitGameRegistry is Operated {
         _;
     }
 
+    /**
+     * @notice Exposes information about exit games quarantine
```

Comment:
`@notice` is used to explain to end user. If we want to use notice here we should probably explain how the whole function works, so the exit game exists part should be expressed as well IMO.

Otherwise we can use `@dev` to make this line like a comment of implementation detail for developer.

https://solidity.readthedocs.io/en/v0.5.10/natspec-format.html